### ISSUE TITLE
Signin/Register page with return url

### ISSUE THREAD

**Issue by marwie** [logto-io/logto#6883]
### What problem did you meet?
Users navigating to the logto signin/signup page do currently not have a way to navigate back to the previous page without using the browser return back navigation

### Describe what you'd like Logto to have
It would be great to be able to pass in a return url to the sign-in/register flow call to the logto client that then appears as a back link on the sign in page (or alternatively is the page that can be opened when clicking the logo).

**Comment by charIeszhao** [comment]
Hi, you can use `postRedirectUri`. For example:

```ts
await signIn({
  redirectUri: 'https://your-app.com/callback',
  postRedirectUri: 'https://your-app.com/previous-page-route'
});
```

**Comment by marwie** [comment]
Hello, thanks for the reply. I'm aware of that and using it already but I'm asking about either a UI element on the sign-in page or about an option to apply the return link to the logo to be clickable.

**Comment by slavaGanzin** [comment]
Yes, I have the same request. For example when you are using WebView in native app there is no way for user to go back.
